CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+Notable changes to this project are documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+Breaking changes (😱!!!):
```

Comment:
We're removing this, right?

Reply:
Yeah, I forgot to remove it from my template before submitting a few test PRs.